File: pset1/mario/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int n, i, a, b;
    //Ask for height, repeat if not an integer
    do
    {
        n = get_int("Height: ");
    }
    while (n < 1);
    //Code for printing the bricks
    for (i = 1; i <= n; i++)
    {
        for (b = n - i; b > 0; b = b - 1)
        {
            printf(" ");
            //Prints blank spaces
        }
        for (a = 1; a <= i; a++)
        {
            printf("#");
            //Prints bricks
        }
        printf("\n");
    }
}

File: pset6/mario/more/mario.py
from cs50 import get_int

height = get_int("Height :")
while height < 1 or height > 8:
    height = get_int("Height :")

for i in range(1, height+1):
    for j in range(height-i, 0, -1):
        print(" ", end="")
    for k in range(1, i+1):
        print("#", end="")
    print("  ", end = "")
    for l in range(1, i+1):
        print("#", end="")
    print()